controllers/ctltools.js

```diff
@@ -97,7 +100,7 @@ const getDeleteHandler = (req, res, next) => {
 };
 
 const getCreateHandler = (req, res, next) => {
-    req.body = sanitizeToolInputs(req.body, true);
+    req.body = sanitizeToolInputs(undefined, req.body, true);
```

Comment:
I do not like the true / false params. But is not part of the PR, only moved. 
But if someone need it, it can be add one method with true/false as parameter and 2 others one for false one for true with speaking names that the code say what is going on.